pytorch_lightning/utilities/parsing.py

```diff
@@ -25,26 +28,28 @@ def str_to_bool(val):
         raise ValueError(f'invalid truth value {val}')
 
 
+def is_picklable(obj: object) -> bool:
```

Comment:
this can be quite a time costly especially for large objects or calling this function quite often...